frontend/src/pages/howToCalculateInstallments/index.js

```diff
@@ -1,12 +1,75 @@
 import React from 'react';
-import { PlaceholderStyle } from '../../components/placeholderStyle/PlaceholderStyleElement';
-import './HowToCalculateInstallmentsElements';
+import { InlineMath } from 'react-katex';
+import {
+  Formula,
+  WhiteSubtitle,
+  Headers,
+  Container,
+  Explanation,
+  FormulaItems,
+  FormulaItem,
+} from './HowToCalculateInstallmentsElements';
+import Footer from '../../components/footerWhite';
+
+import { SemiHeader } from '../../components/semiHeader/SemiHeader';
+import { PurpleAccentParagraph } from '../../components/AccentParagraph/AccentParagraph';
+import 'katex/dist/katex.min.css';
 
 function HowToCalculateInstallments() {
+  const fixedFormula = '\\frac{A \\ast b}{m*(1-(\\frac{m}{m+b})^n)}';
+  const decrementingFormula =
+    '\\frac{A}{n} \\ast (1+(n-i-1) \\ast \\frac{b}{m})';
+  const descendingNames = [
+    'i - numer miesiąca, dla którego obliczana jest rata',
+    'A – kwota udzielonego kredytu',
+    'b – wysokość oprocentowania kredytu w skali roku',
+    'm – liczba miesięcy w roku',
+    'n – liczba rat',
+  ];
+  const fixedNames = [...descendingNames];
+  fixedNames.shift();
```

Comment:
It is fine, but just a note. You can also use this trick to get the first element:
```js
const [firstElement,] = descendingNames;
```

If you needed first element and the rest in separate variable, then:

```js
const [firstElement, ...fixedNames] = descendingNames;
```

Or just what you actually do:
```js
const [, ...fixedNames] = descendingNames;
```

Reply:
thanks!